dashboard/app/models/programming_class_method.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: programming_class_methods
+#
+#  id                   :bigint           not null, primary key
+#  programming_class_id :integer
+#  key                  :string(255)
+#  name                 :string(255)
+#  content              :text(65535)
+#  parameters           :text(65535)
+#  examples             :text(65535)
+#  syntax               :string(255)
+#  external_link        :string(255)
+#  created_at           :datetime         not null
+#  updated_at           :datetime         not null
+#
+# Indexes
+#
+#  index_programming_class_methods_on_key_and_programming_class_id  (key,programming_class_id) UNIQUE
+#
+class ProgrammingClassMethod < ApplicationRecord
```

Comment:
Is the "class" in the name to make it clear that it's part of the "class" hierarchy and the "expression" hierarchy?  (This name feels slightly clunky but I can't think of anything better. :-)

Reply:
Maybe ProgrammingMethod would be descriptive enough?  (Just throwing this out as a suggestion, I don't feel strongly about it either way.)